doc/whats_new.rst

```diff
@@ -27,6 +27,7 @@ Bug
 - Allow files with no stim channel, which could be the case for example in resting state data, by `Mainak Jas`_ (`#167 <https://github.com/mne-tools/mne-bids/pull/167/files>`_)
 - Better handling of unicode strings in TSV files, by `Mainak Jas`_ (`#172 <https://github.com/mne-tools/mne-bids/pull/172/files>`_)
 - Fix separator in scans.tsv to always be `/`, by `Matt Sanderson`_ (`#176 <https://github.com/mne-tools/mne-bids/pull/176>`_)
+- Allow raw data from KIT systems to have two marker files specified, by `Matt Sanderson`_ (`#167 <https://github.com/mne-tools/mne-bids/pull/173>`_)
```

Comment:
two? or multiple? it can have more than two, no?

Reply:
I guess two is right :-)

> Marker coil files MUST have an acquisition parameter if there is more than
one file, and no more than two files may be associated with the one raw file.

see: https://github.com/bids-standard/bids-specification/pull/62/files

Reply:
yeah, the idea is that it would be fixed to not allow more than 2 markers to be associated with the one .con/.sqd file

Reply:
can you move this to the changelog of v0.3?

Reply:
+1, moving this to the changelog of `current` (at top of file)

Reply:
sorry for the delay! It has been moved :)